Sources/JExtractSwift/Swift2JavaTranslator+JavaThunkPrinting.swift

```diff
@@ -0,0 +1,717 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Re filename: I don't think we should be calling the Java side "thunks", if we're looking for names we've been calling them accessors I guess. I'd like to keep that distinction

Reply:
That sounds good! Let's start using "bindings" consistently then for the Java generated side 👍